de.dlr.sc.virsat.model.extension.requirements.ui/src/de/dlr/sc/virsat/model/extension/requirements/ui/wizard/ReqIfMappingPage.java

```diff
@@ -0,0 +1,232 @@
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package de.dlr.sc.virsat.model.extension.requirements.ui.wizard;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.eclipse.emf.common.util.EList;
+import org.eclipse.jface.wizard.WizardPage;
+import org.eclipse.rmf.reqif10.Specification;
+import org.eclipse.rmf.reqif10.impl.ReqIFImpl;
+import org.eclipse.swt.SWT;
+import org.eclipse.swt.custom.CCombo;
+import org.eclipse.swt.custom.TableEditor;
+import org.eclipse.swt.events.SelectionEvent;
+import org.eclipse.swt.events.SelectionListener;
+import org.eclipse.swt.layout.GridData;
+import org.eclipse.swt.layout.GridLayout;
+import org.eclipse.swt.widgets.Composite;
+import org.eclipse.swt.widgets.Table;
+import org.eclipse.swt.widgets.TableColumn;
+import org.eclipse.swt.widgets.TableItem;
+
+import de.dlr.sc.virsat.model.dvlm.Repository;
+import de.dlr.sc.virsat.model.dvlm.structural.StructuralElementInstance;
+
+/**
+ * A wizard page to review the requirement type which is used for imports
+ *
+ */
+public class ReqIfMappingPage extends WizardPage implements SelectionListener {
+
+	private static final String PAGE_TITLE = "ReqIF Mapping";
+	
+	private static final String COLUMN_SPECIFICATION_LABEL = "Specification";
+	private static final String COLUMN_CONTAINER_LABEL = "Container";
+	
+	private static final int COLUMN_NAME_WIDTH = 300;
+	private static final int COLUMN_TYPE_WIDTH = 300;
+
+
+	private static final int SPEC_COLUMN_INDEX = 1;
+
+	private Table table;
+	private List<TableItem> tableItems = new ArrayList<TableItem>();
+	private List<CCombo> editors = new ArrayList<CCombo>();
+	
+	private ReqIFImpl reqIfContent;
+
+	private Repository repository;
+	/**
+	 * Constructor
+	 */
+	protected ReqIfMappingPage() {
+		super(PAGE_TITLE);
+		setTitle(PAGE_TITLE);
+		setDescription("Check the specification elements to import and their container element in the system model.");
+	}
+
+
+	@Override
+	public void createControl(Composite parent) {
+
+		Composite content = new Composite(parent, SWT.NONE);
+		content.setLayout(new GridLayout());
+		content.setLayoutData(new GridData(GridData.FILL_BOTH | GridData.GRAB_HORIZONTAL | GridData.GRAB_HORIZONTAL));
+		setControl(content);
+
+		table = new Table(content, SWT.CHECK | SWT.BORDER | SWT.H_SCROLL);
+		table.setLayout(new GridLayout());
+		table.setLayoutData(new GridData(GridData.FILL_BOTH | GridData.GRAB_HORIZONTAL | GridData.GRAB_VERTICAL));
+		table.setHeaderVisible(true);
+		table.addSelectionListener(this);
+
+		TableColumn columnColumnName = new TableColumn(table, SWT.NULL);
+		columnColumnName.setWidth(COLUMN_NAME_WIDTH);
+		columnColumnName.setText(COLUMN_SPECIFICATION_LABEL);
+
+		TableColumn columnType = new TableColumn(table, SWT.NONE | SWT.DROP_DOWN);
+		columnType.setText(COLUMN_CONTAINER_LABEL);
+		columnType.setWidth(COLUMN_TYPE_WIDTH);

```

Comment:
Maybe a more spelling name, e.g. `columnSpecification` and `columnContainer` ? Then also rename the width constants?